### ISSUE TITLE
py 2.4 has no ssl support

### ISSUE THREAD

**Issue by fulv** [gforcada/ansible-compile-python#7]
Unfortunately, the python 2.4 that results from this role is still not quite usable for Plone purposes.
For example, if you create a 2.4 virtualenv and then run `python bootstrap.py`, you get a bunch of errors such as:

```
Download error: unknown url type: https -- Some packages may not be found!
Couldn't find index page for 'zc.buildout' (maybe misspelled?)

[etc]
```

I tried to change the `./configure` step in the playbook by using the tip from the docs (http://docs.plone.org/manage/troubleshooting/exceptions.html#ioerror-errno-url-error-unknown-url-type-https):

```
./configure --with-package=_ssl
```

But that did not help, either.

So in the end, in order to deploy a Plone 3 site, I always revert to the general buildout for python (`collective/buildout.python`), which does not have these problems.  I have not figured out what the difference is between `collective/buildout.python` and this one for 2.4.

@gforcada do you have any ideas?

**Comment by gforcada** [comment]
@fulv maybe the ssl headers are missing?

**Comment by gforcada** [comment]
I'm closing this as a `won't fix`, python 2.4 is maybe way too complex and a corner case to handle.
